src/course-updates/CourseUpdates.test.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { initializeMockApp } from '@edx/frontend-platform';
+import MockAdapter from 'axios-mock-adapter';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+import CourseUpdates from './CourseUpdates';
+import messages from './messages';
+import initializeStore from '../store';
```

Comment:
Can we follow a uniform approach of grouping local imports from nearest to the ballistic location?

```suggestion
import initializeStore from '../store';
import CourseUpdates from './CourseUpdates';
import messages from './messages';
```

Reply:
Done